java_src/foam/core/AbstractFObject.java

```diff
@@ -63,11 +63,18 @@ public int compareTo(Object o) {
 
   public FObject setProperty(String prop, Object value) {
     PropertyInfo property = ((PropertyInfo) getClassInfo().getAxiomByName(prop));
+    if (property == null) {
```

Comment:
This will also break users that chain setProperty() calls because this method is expected to return 'this'.

if ( property != null ) property.set(this, value);

return this;